mne/event.py

```diff
@@ -210,6 +210,11 @@ def read_events(filename, include=None, exclude=None, mask=0):
     mask : int | None
         The value of the digital mask to apply to the stim channel values.
         The default value is 0. ``None`` skips masking.
```

Comment:
`None` should be made the default now. That way it does nothing for both `mask_type`s.